util/util.go

```diff
@@ -207,6 +208,36 @@ func RunTemplate(templateContent string, data interface{}, out io.Writer) error
 		"trimSuffix": func(content, suffix string) string {
 			return strings.TrimSuffix(content, suffix)
 		},
+		"cwd": func() (string, error) {
+			return os.Getwd()
+		},
+		"findLatestArtifact": func(content string) (string, error) {
+			matches, err := doublestar.Glob(content)
+			if err != nil {
+				return "", err
+			}
+			artifact := struct {
+				Path string
+				Info os.FileInfo
+			}{}
+			for _, matchPath := range matches {
+				info, err := os.Stat(matchPath)
+				if err != nil {
+					return "", err
+				}
+				if artifact.Path == "" {
+					artifact.Info = info
+					artifact.Path = matchPath
+				} else if artifact.Info.ModTime().Before(info.ModTime()) {
+					artifact.Info = info
+					artifact.Path = matchPath
+				}
+			}
+			if artifact.Path == "" {
+				return "", fmt.Errorf("No Artifact Found matching glob: %v", content)
```

Comment:
`%v` => `%s` I think here, not sure it matters but it is guaranteed to be a string here.